File: Leetcode/JS/Leetcode1816-截断句子.js
var truncateSentence = function(s, k) {
    let res = "";
    for(let i = 0;i < s.length && k != 0;i++){
        if(s[i] == ' ') k--;
        if(k) res += s[i];
    }
    return res;
};
// var truncateSentence = function(s, k) {
//     let arr = s.split(' ')
//     arr = arr.slice(0,k);
//     return arr.join(' ');
// };

File: Leetcode/sbgo/Leetcode1816-截断句子.go
package sbgo

func truncateSentence(s string, k int) string {
	res := []byte{}
	for i := 0; i < len(s) && k != 0; i++ {
		if s[i] == ' ' {
			k--
		}
		if k != 0 {
			res = append(res, s[i])
		}
	}
	return string(res)
}
